modules/olm-configuring-aws-sts.adoc

```diff
@@ -0,0 +1,182 @@
+// * storage/container_storage_interface/rosa-persistent-storage-aws-efs-csi.adoc
+
+////
+ifeval::["{context}" == "olm-adding-operators-to-a-cluster"]
+:olm-generic:
+endif::[]
+ifeval::["{context}" == "persistent-storage-csi-aws-efs"]
+:known-operator:
+endif::[]
+ifeval::["{context}" == "osd-persistent-storage-aws-efs-csi"]
+:known-operator:
+endif::[]
+ifeval::["{context}" == "persistent-storage-csi-aws-efs"]
+:known-operator:
+endif::[] 
+////
+
+:_content-type: PROCEDURE
+[id="olm-configuring-aws-sts_{context}"]
+= Configuring an Operator to use Amazon Web Services Secure Token Service
+
+You can use the CCO utility (`ccoctl`) to create resources for Operators on Amazon Web Services (AWS) clusters that use the AWS Secure Token Service (STS). Operators that use STS assign components IAM roles that provide short-term, limited-privilege security credentials. For more information, see "About manual mode with AWS Secure Token Service".
+
+Perform this procedure after installing an Operator that can use STS, but before installing any required drivers for the Operator. If you perform this procedure after installing a driver for an Operator, the Operator might not function correctly.
+
+[NOTE]
+====
+By default, `ccoctl` creates objects in the directory in which the commands are run. To create the objects in a different directory, use the `--output-dir` flag. This procedure uses `<path_to_ccoctl_output_dir>` to refer to this directory.
+
+Some `ccoctl` commands make AWS API calls to create or modify AWS resources. You can use the `--dry-run` flag to avoid making API calls. Using this flag creates JSON files on the local file system instead. You can review and modify the JSON files and then apply them with the AWS CLI tool using the `--cli-input-json` parameters.
+====
+
+.Prerequisites
+
+* You have an AWS cluster that is configured to use STS.
+
+* You have installed an Operator that can use STS.
+
+.Procedure
+
+. Configure the `ccoctl` utility:
+
+.. Get the CCO container image from the {product-title} release image by running the following command:
++
+[source,terminal]
+----
+$ CCO_IMAGE=$(oc adm release info --image-for='cloud-credential-operator')
+----
+
+.. Extract the `ccoctl` binary from the CCO container image within the {product-title} release image by running the following command:
++
+[source,terminal]
+----
+$ oc image extract $CCO_IMAGE --file="/usr/bin/ccoctl" -a ${PULL_SECRET_FILE}
+----
+//Need to explain/define PULL_SECRET_FILE I think
+
+.. Change the permissions to make `ccoctl` executable by running the following command:
++
+[source,terminal]
+----
+$ chmod 775 ccoctl
+----
+////
+//This strat may not actually make sense due to how STS is implemented and given the move away from minting.
+ifdef::known-operator[]
+. Determine the name of the `CredentialsRequest` custom resource (CR) in the `openshift-cloud-credential-operator` namespace for the Operator: #How do they do that?#
```

Comment:
For now, I have mentioned it in the intro of the EFS CSI version to avoid messing with the rest of that installation content (out of scope and I am also concerned about messing it up). If this all look ok though I will raise it with Lisa to see if she can advise.